### DISCUSSION TITLE
Brand change with AIO

### DISCUSSION THREAD

**Discussion by bvwgti** [harman-f/MHI2_MIB2_AIO_FW_Update_Template#8] [Q&A]
Can the AIO firmware be used for brand changes (possibly with additional changes to metainfo2) or do you need to stay within the same brand's FW?

**Comment by harman-f** [comment]
what exactly do you want to achive?

**Reply by bvwgti** [reply]
I have a VW and am hoping to be able to retrofit the 9.2" display. My current main unit can't support the 9.2"display and can't be upgraded, so I am looking at sourcing a used MIB2 High. It may be easier to source a non-VW main unit; therefore, I was wondering if I could use the AIO firmware to convert the main unit to a VW main unit?

**Reply by harman-f** [reply]
Which unit do you currently have in your car?
Not a HArman high I assume?

Brand conversions are possible, but need some extra tweaks to go through fine, which are not explained here in detail.
